docs/local-dev-setup.md

```diff
@@ -10,6 +10,7 @@ Follow these steps to set up your local development environment for working with
 - Make sure that you have a current version of Python 3 and pip.
 - Make sure that a recent stable version of [Go is installed](https://go.dev/doc/install) and available in your PATH environment variable.
 - Make sure the [Brew](https://brew.sh) is installed
+- Make sure that [curl](https://curl.se/) and unzip are installed.
```

Comment:
@vburckhardt We don't have anything in the docs that's mentions curl. What process needs that? (Also requires an install only for Windows.)

Also no mention of extracting files. What do we need an unzip tool for (also already included by default for windows and macOS).

Reply:
Some confusion here I think..
We dont need curl or unzip for development. However they are needed to run the migration script that is in this PR https://github.com/terraform-ibm-modules/common-dev-assets/pull/85

So I think we should remove this line. Brew is valid right now, as we install checkov with brew

Reply:
Thanks @ocofaigh. Looks like that migration script renders redundant most of the migration content I just pushed in #12.

Reply:
@ocofaigh - curl and unzip are needed by install-deps.sh which is part of the regular dev workflow, so these are valid here.

Reply:
oh yes, valid point!